### PR TITLE
Fix broken Z-Wave link

### LINKED ISSUE #26874: Link issue on page: https://www.home-assistant.io/integrations/zwave_js/
### Feedback

On the page:
https://www.home-assistant.io/integrations/zwave_js/.

The link to:
https://www.home-assistant.io/integrations/zwave_js/integrations/zwave_js/#which-z-wave-controller-should-i-buy

That is contained (under link text "here") within the paragraph:
A [supported Z-Wave controller](https://www.home-assistant.io/docs/z-wave/controllers/#supported-z-wave-usb-sticks--hardware-modules). First-time user? For recommendations on what to buy, go [here](https://www.home-assistant.io/integrations/zwave_js/integrations/zwave_js/#which-z-wave-controller-should-i-buy).

Results in : 
Oh no! This page does not exist 😞

Perhaps it should link to the previous page in the paragraph or maybe somewhere else or the target page needs to be modified? It is not clear what the correct solution would be.

### URL

https://www.home-assistant.io/integrations/zwave_js/

### Version

2023.3.6

### Additional information

_No response_

### LINKED ISSUE #26952: Broken link in documentation
### Feedback

link is broken: if you click on 'here'

A [supported Z-Wave controller](https://www.home-assistant.io/docs/z-wave/controllers/#supported-z-wave-usb-sticks--hardware-modules). First-time user? For recommendations on what to buy, go [here](https://www.home-assistant.io/integrations/zwave_js/integrations/zwave_js/#which-z-wave-controller-should-i-buy).

### URL

https://www.home-assistant.io/integrations/zwave_js/

### Version

2023.4.2

### Additional information

_No response_

### PR DESCRIPTION
**Pull request by c0ffeeca7**
- closes #26952
- closes #26874

## Type of change

- [x] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

**Comment by c0ffeeca7** [comment]
Thank you for the catch, @PickleRick404, @chibiconsulting :+1:

File: source/_integrations/zwave_js.markdown [hunk 1]
```diff
@@ -50,7 +50,7 @@ The Z-Wave integration allows you to control a Z-Wave network via the [Z-Wave JS
 
 To run a Z-Wave network, you need the following elements:
 
-* A [supported Z-Wave controller](/docs/z-wave/controllers/#supported-z-wave-usb-sticks--hardware-modules). First-time user? For recommendations on what to buy, go [here](integrations/zwave_js/#which-z-wave-controller-should-i-buy).
+* A [supported Z-Wave controller](/docs/z-wave/controllers/#supported-z-wave-usb-sticks--hardware-modules). First-time user? For recommendations on what to buy, go [here](/integrations/zwave_js/#which-z-wave-controller-should-i-buy).
```
Comment (MartinHjelmare) [inline source/_integrations/zwave_js.markdown]: Links to anchors on the same page don't need the full path.
```suggestion
* A [supported Z-Wave controller](/docs/z-wave/controllers/#supported-z-wave-usb-sticks--hardware-modules). First-time user? For recommendations on what to buy, go [here](#which-z-wave-controller-should-i-buy).
```

**Review by MartinHjelmare — CHANGES_REQUESTED** [review]

File: source/_integrations/zwave_js.markdown (hunk 1 above)
Comment (c0ffeeca7) [reply source/_integrations/zwave_js.markdown]: Thank you, @MartinHjelmare! :+1:

**Review by MartinHjelmare — APPROVED** [review]
Thanks!

**Pull request merged**